docfx/articles/core/installation.md

```diff
@@ -1,20 +1,11 @@
 ## Installation
 
-You can get VR Builder from the [Unity Asset Store](https://assetstore.unity.com/packages/tools/visual-scripting/vr-builder-201913) or from our [GitHub](https://github.com/MindPort-GmbH/VR-Builder/releases). In the first case, you will be able to download and import the VR Builder package from the package manager. If you have downloaded a .unitypackage from GitHub, just import it in the project.
+You can get VR Builder from the [Unity Asset Store](https://u3d.as/3pUD) or from our [GitHub](https://github.com/MindPort-GmbH/VR-Builder/releases). In the first case, you will download and import the VR Builder package from the Package Manager. From GitHub, you can either download a released .UnityPackage or pull the repository and import the 
+`package.json` file in the Package Manager.
```

Comment:
Are you shre that "pull the repository and import the 
`package.json` file in the Package Manager." works? Isn't that waht cousingthis bug https://mindport.atlassian.net/browse/MP-2969?

Reply:
I mean pointing to a local package, not entering a Git URL in the Package Manager, which is what the bug talks about.

Reply:
I understood I wasn't sure if it would result in the same issue.

Reply:
No it's not the same, a local package works fine.